Teapot/MockTeapot.swift

```diff
@@ -8,7 +9,12 @@ open class MockTeapot: Teapot {
         case missingMockFile(String)
         case invalidMockFile(String)
     }
-
+    
+    /// Initialiser.
+    ///
+    /// - Parameters:
+    ///   - baseURL: baseURL for the call, used to call the super init.
```

Comment:
This isn't clear to me, also I think this is never actually used, if that's the case, can't we omit it from the init method here and just send a dummy value that works inside the method?